core/domain/suggestion_services_test.py

```diff
@@ -911,13 +961,63 @@ def test_accept_suggestion_commit_message_after_updating_a_suggestion(self):
             'content', '<p>old content html</p>').to_dict()
         exploration.states['state 1'].update_content(
             state_domain.SubtitledHtml.from_dict(old_content))
-        exp_services._save_exploration(self.author_id, exploration, '', [])  # pylint: disable=protected-access
+        change_list = [exp_domain.ExplorationChange({
```

Comment:
Prefer not having to repeat code, if possible lets make this a utility as mentioned above so that we can use this across all tests.